### DISCUSSION TITLE
Can we remove Wagtail cache package?

### DISCUSSION THREAD

**Discussion by frjo** [OpenTechFund/opentech.fund#3999] [General]
It is used on a single model `ApplyHomePage`.

There is no MIDDLEWARE configured for it so unsure if it is even active.

These could then be removed:

* wagtail-cache
* wagtail-purge

**Comment by theskumar** [comment]
I think it should be good to remove, it was primarily used on the public site to cache the rendered wagtail pages.

**Comment by theskumar** [comment]
Curious if we can starting the GitHub's Discussion for these type of issues and once the have an outcome it can be converted to issues, thought? 

![Screenshot 2024-06-20 at 6  12 53@2x](https://github.com/HyphaApp/hypha/assets/236356/033c535f-8807-4d7a-9007-1d9d769c72a3)

**Comment by theskumar** [comment]
OR maybe it's good much! :D

**Comment by theskumar** [comment]
There is also an option to converted this issue into a discussion nice !

![Screenshot 2024-06-20 at 6  15 28@2x](https://github.com/HyphaApp/hypha/assets/236356/2a2e06e8-b898-40c3-8a1b-a48805df28b1)

**Comment by theskumar** [comment]
I also found this https://github.com/HyphaApp/hypha/discussions/2330. Maybe a good time to close GitHub Discussions? 

_PS: haven't found discourse being used so for technical discussion, but I see how discourse will be a better tool to discussion product feature related discussions, releases, etc._
